[DOCS] Adding PowerShell version of the setupvars and build_samples scripts for master (#22631)
openvinotoolkit/openvino: docs/articles_en/get_started/installing-openvino-overview/installing-openvino-windows-header/installing-openvino-from-archive-windows.rst

Step 2: Configure the Environment
+++++++++++++++++++++++++++++++++

You must update several environment variables before you can compile and run OpenVINO™ applications.

.. tab-set::

   .. tab-item:: PowerShell
      :sync: powershell

      Open the PowerShell, and run the ``setupvars.ps1`` file to temporarily set your environment variables.

      .. code-block:: sh

         . <path-to-setupvars-folder>/setupvars.ps1

   .. tab-item:: Command Prompt
      :sync: cmd

>       Open the Command Prompt, and run the ``setupvars.bat`` batch file to temporarily set your environment variables.

Reviewer: ```suggestion
      Open the CMD, and run the ``setupvars.bat`` batch file to temporarily set your environment variables.
```

      If your ``<INSTALL_DIR>`` is not ``C:\Program Files (x86)\Intel\openvino_2023``, use the correct directory instead.

      .. code-block:: sh

         "C:\Program Files (x86)\Intel\openvino_2023\setupvars.bat"

      .. important::

>          The above command must be re-run every time a new Command Prompt window is opened.

Reviewer: ```suggestion
         You need to run the command for each new CMD window.
```
